roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh

```diff
@@ -80,6 +80,10 @@ EOF
       NEEDS_RESTART=1
     fi
 
+    if ! `systemctl -q is-active dnsmasq.service`; then
```

Comment:
Does this work consistently between all "supported" releases? RHEL 7.1, Fedora 24/25?